library/src/lib/calendar/calendar-views/calendar-year-view/calendar-year-view.component.ts

```diff
@@ -17,6 +16,11 @@ import { Subject } from 'rxjs';
 })
 export class CalendarYearViewComponent implements AfterViewChecked, OnInit, OnDestroy {
 
+    /** @hidden
+     *  This variable is used to define which year should be possible to focus (tabindex = 0)
+     * */
+    activeYear: number;
```

Comment:
I don't understand the purpose of this just by reading the docs comment, consider improving the description a bit

Reply:
Actually I don't really get what this even does

Reply:
It defines which year can be focused by tab key. I could be a little unclear. Description changed.